homeassistant/components/nina/entity.py

```diff
@@ -0,0 +1,19 @@
+"""Entity for the nina integration."""
+from dataclasses import dataclass
+
+
+@dataclass
+class NinaWarningData:
```

Comment:
I'm not following how this ended up in the entity module. Could you explain your reasoning for this one?

Reply:
No, you are absolutely right. This data class is used only by the coordinator, so I moved it there. Beside this the entity module would be indeed wrong.